docker/formbricks.sh

```
@@ -282,62 +335,282 @@ EOT
+
+    # Wait for MinIO to be ready
+    echo "⏳ Waiting for MinIO to start..."
+    for i in {1..30}; do
+        if docker run --rm --network formbricks_default \
+            -e MC_HOST_minio=http://$minio_root_user:$minio_root_password@minio:9000 \
+            minio/mc:latest \
+            admin info minio >/dev/null 2>&1; then
+            echo "✅ MinIO is ready!"
+            break
+        fi
+        echo "⏳ Waiting... ($i/30)"
```

Shellcheck (0.10.0)

[warning] i is referenced but not assigned.

(SC2154)